corehq/toggles.py

```diff
@@ -1827,3 +1827,11 @@ def _commtrackify(domain_name, toggle_is_enabled):
     [NAMESPACE_DOMAIN],
     help_link='https://confluence.dimagi.com/display/ccinternal/Two-Stage+Mobile+Worker+Account+Creation',
 )
+
+ICDS_BIHAR_DEMOGRAPHICS_API = StaticToggle(
```

Comment:
For future PRs, it's helpful to stick new flags in the middle of the file - we get merge conflicts when everybody is adding flags to the end of the file.